apps/frontend/src/hooks/zero/useGetOpenTrove.ts

```diff
@@ -23,13 +26,13 @@ export const useGetOpenTrove = () => {
     [account],
   );
 
-  const { loading, data } = useQuery(GetUserOpenTroveDocument, {
+  const { loading, data } = useGetUserOpenTroveQuery({
     variables: troveConfig,
     client: zeroClient,
   });
 
   useEffect(() => {
-    if (data && data.trove && !loading) {
+    if (data?.trove?.status === TroveStatus.Open && !loading) {
       setIsTroveOpenExists(!!data.trove);
     }
   }, [loading, data]);
```

Comment:
I think rewriting it to useMemo would be cleaner as it would remove useState and useEffect